### DISCUSSION TITLE
New sets not showing up?

### DISCUSSION THREAD

**Discussion by pewingfield** [leolelego/brickie.swiftui#106] [Q&A] [answered]
I’ve been trying to manage my collection and some upcoming releases, and it looks like sets at least a few weeks old are still showing as ??????

**Comment by leolelego** [comment, marked as answer]
Hi, did they shows up on the brickset website ? 
Can you try to logout and login back in the app, could be a cache issue.

**Reply by pewingfield** [reply]
That seems to have been the fix. Should maybe add a “refresh cache” button in settings

**Reply by leolelego** [reply]
@perfecteagle this will be in the next release 1.5.2 available in february someday. try to make little iteration to solve small issues, intead of big work.
